draft-ietf-quic-ack-frequency.md

```diff
@@ -339,17 +333,32 @@ send an acknowledgement immediately on receiving a reordered ack-eliciting
 packet. This extension modifies this behavior.
 
 If the endpoint has not yet received an ACK_FREQUENCY frame, or if the most
-recent frame received from the peer has an `Ignore Order` value of `false`
-(0x00), the endpoint MUST immediately acknowledge any subsequent packets that
+recent frame received from the peer has a `Reordering Threshold` value that is
+not 0x00, the endpoint MUST immediately acknowledge any subsequent packets that
 are received out of order.
```

Comment:
```suggestion
smaller than the number of outstanding unacknowledged packets since
an on-going reordering started, the endpoint MUST immediately send an
ACK frame.
```